lambda/MAS/Services/S3Service.cs

```diff
@@ -1,20 +1,18 @@
-﻿using Amazon.Lambda.Core;
-using Amazon.S3;
+﻿using Amazon.S3;
 using Amazon.S3.Model;
 using MAS.Configuration;
 using MAS.Models;
 using Microsoft.Extensions.Logging;
 using System;
-using System.Text;
 using System.Threading.Tasks;
 
 namespace MAS.Services
 {
-    public interface IS3Service
+    public interface IStaticContentService
     {
-        Task<PutObjectResponse> WriteToS3(Item item);
+        Task<PutObjectResponse> WriteToS3(string slug, string body);
```

Comment:
We've renamed the interface to IStaticContentService which is great, but the method should be S3 agnostic too, e.g. just `Write` or something